src/doc/unstable-book/src/language-features/default-field-values.md

```diff
@@ -0,0 +1,89 @@
+# `default_field_values`
+
+The tracking issue for this feature is: [#132162]
+
+[#132162]: https://github.com/rust-lang/rust/issues/132162
```

Comment:
Question: should we also link the RFC directly here to save a layer of indirection?